sos/report/plugins/ceph_common.py

```diff
@@ -117,6 +114,87 @@ def setup(self):
             self.add_cmd_output([f"microceph {cmd}" for cmd in cmds],
                                 subdir='microceph')
 
+            dqlite_crt = "/var/snap/microceph/common/state/cluster.crt"
+            self.add_cmd_output(
+                f"openssl x509 -in {dqlite_crt} -noout -dates",
+                subdir="microceph"
+            )
+
+            db_path = "/var/snap/microceph/common/state/database"
+
+            # Check for inconsistent dqlite db intervals
+            self.add_dir_listing(
+                db_path,
+                suggest_filename="ls_microceph_dqlite_dir",
+                subdir="microceph",
+            )
+
+            self.add_copy_spec(
+                [
+                    f"{db_path}/info.yaml",
+                    f"{db_path}/cluster.yaml",
+                    f"{db_path}/../daemon.yaml",
+                ]
+            )
+
+            queries = [
+                {
+                    "query": (
+                        "SELECT * FROM sqlite_master WHERE type=\"table\";"
+                    ),
+                    "suggested_file_suffix": "schema",
+                },
+                {
+                    "query": (
+                        "SELECT * FROM config WHERE NOT ( "
+                        "key LIKE \"%keyring%\" OR "
+                        "key LIKE \"%ca_cert%\" OR "
+                        "key LIKE \"%ca_key%\" );"
+                    ),
+                    "suggested_file_suffix": "config",
+                },
+                {
+                    "query": "SELECT * FROM services;",
+                    "suggested_file_suffix": "services",
+                },
+                {
+                    "query": (
+                        "SELECT id, name, expiry_date "
+                        "FROM core_token_records;"
+                    ),
+                    "suggested_file_suffix": "token_records",
```

Comment:
I'm not sure that we need the token records; these tend to be short lived (they expire after 3 hrs by default) so the table will almost always be empty.

Reply:
I'm amenable to that argument; this is fine then :)